src/tt_compat.cpp

```diff
@@ -0,0 +1,36 @@
+#include <lua.hpp>
+#include <lauxlib.h>
+#include <string>
+#include "tt_compat.h"
+
+char* SER_MARKER_MAP = (char*)"map";
+char* SER_MARKER_SEQ = (char*)"seq";
+
+// Variable is needed to avoid collisions in LUA_REGISTRY.
+static const char TT_NULL_ADDR = 't';
```

Comment:
Can you explain why random choice better then a constant big number?

Reply:
This code is actually taken from the lua manual: https://www.lua.org/pil/27.3.1.html, the method is called "bulletproof".
"The C link editor ensures that this key is unique among all libraries.".

We can also just use a complex key like "__lua_simdjson__null_addr" to achieve merely the same result(to avoid collision possibility).